includes/documents/abstract-wcpdf-order-document.php

```diff
@@ -817,17 +817,19 @@ public function has_header_logo() {
 	/**
 	 * Return logo id
 	 *
-	 * @return ?int
+	 * @return int
 	 */
-	public function get_header_logo_id(): ?int {
+	public function get_header_logo_id(): int {
 		$header_logo_id = 0;
 
 		if ( ! empty( $this->settings['header_logo'] ) ) {
 			$attachment_id  = $this->get_settings_text( 'header_logo', $header_logo_id, false );
 			$header_logo_id = is_numeric( $attachment_id ) ? absint( $attachment_id ) : 0;
 		}
+		
+		$header_logo_id = apply_filters( 'wpo_wcpdf_header_logo_id', $header_logo_id, $this );
 
-		return apply_filters( 'wpo_wcpdf_header_logo_id', $header_logo_id, $this );
+		return $header_logo_id ?? 0;
```

Comment:
Would it be good also to check if the value is numeric in case the filter returns any other types?